lib/memcached.js

```diff
@@ -913,7 +913,7 @@ Client.config = {
     value = Utils.escapeValue(value);
 
     length = Buffer.byteLength(value);
-    if (length > this.maxValue) {
+    if (length + fullkey.length + 71 > this.maxValue) {
```

Comment:
Documentation I found said that 71 is the length of the metadata that memcache uses to store information about each key value pair. That may need to be adjusted or set as a system variable.